### ISSUE TITLE
cryptoknight lite v7 for bbs not working at cryptoknight:19932

### ISSUE THREAD

**Issue by jch9678** [RainbowMiner/RainbowMiner#462]
I only get rejects.

**Comment by jch9678** [comment]
stellitev4 not working at stellite.miner.rocks:5005
all rejects

**Comment by jch9678** [comment]
cryptonight arto barely working at arto.ingest.cryptoknight.cc:51202
started out with lots of accepted at a fast pace but then got lots of compute errors

**Comment by jch9678** [comment]
These are on srbminer 1.8  I tried to watch them all re-benchmark but may have missed one that didn't work.

Sorry for the list

**Comment by RainbowMiner** [comment]
Thank you for the list! I will take care of that. Would it be possible, that you drop the current debug file? Then I will try to go through all of them, step by step.

**Comment by jch9678** [comment]
[debug-2019-03-08.zip](https://github.com/RainbowMiner/RainbowMiner/files/2946435/debug-2019-03-08.zip)

Here it is.

**Comment by jch9678** [comment]
haven is not working at minerrocks
The error I get is The Specified Class Was Not Found

ERROR : The specified class was not found.
[2019-03-10 13:32:15] Reconnecting to ca.haven.miner.rocks:4005 4005 4005 in 10 seconds

**Comment by RainbowMiner** [comment]
Oh dear! I see. There is a problem in the port preselection. It will be fixed with v3.9.9.3, just now.

**Comment by RainbowMiner** [comment]
Should be fixed, by now.

**Comment by jch9678** [comment]
I don't believe they are.  I updated machines from 3.9.9.3 from a few hours ago and I've already had to exclude some algos on 3 different machines.

**Comment by jch9678** [comment]
I guess what I need to do is take one machine and delete the stat file for each of the cryptonight algos and let them re-benchmark and record my observations.

**Comment by RainbowMiner** [comment]
Oh. I have been too fast with closing this issue. So, if you really find the time and would record your observations on the cryptonight algorithms, I would be more than happy!

**Comment by jch9678** [comment]
No problem

I'll report back when I get around to it. In the meantime I've just
disabled minerrocks and cryptoknight pools

**Comment by jch9678** [comment]
I erased all 13 srbminer stats on a test machine and let it benchmark. I'm not sure why but it only benchmarked 9 algos. Here is what I found so far:

--CryptoNightR_ - Works
--CryptoNightLiteV7_ - cryptoknight,19932, does not work
--CryptoNightXao_ - alloy,cryptoknight,5662, does not work, invalid share diff
--CryptoNightV8_ - v8 lethean,cryptoknight,8882? not sure if v8 works, pool requested change to v4 and        srbminer changed to v4 so I'm not sure what was being mined.
--CryptoNightHeavyXhv_ - haven.minerrocks,4005, works 1 share accepted
--CryptoNightV7_ - v7,cryptoknight,9222, no shares, probably works, maybe not enough time
--CryptoNightHeavy_ - heavy,nicehash, no share, probably works, needs more time at nicehash
--CryptoNightHeavyTube_ - bittube,minerrocks,5555, works
--CryptoNightRto_ - arto,cryptoknight,51202, works but then get compute errors, closed srbminer, benchmarked again, same result

Did not benchmark:
CryptoNightFast_
CryptoNightReverseWaltz_
CryptoNightSuperFast_
CryptoNightXtl_
